src/AppInstallerCLICore/ExecutionArgs.h

```diff
@@ -162,7 +162,11 @@ namespace AppInstaller::CLI::Execution
             Max
         };
 
-        bool Contains(Type arg) const { return (m_parsedArgs.count(arg) != 0); }
+        template<typename... T, std::enable_if_t<(... && std::is_same_v<T, Args::Type>), bool> = true>
+        bool Contains(T... arg) const

```

Comment:
You could also use a `std::initializer_list` with the low cost of putting `{}` around your parameter list.  But it definitely wouldn't be as fun as figuring out this.